src/openms/source/FORMAT/AbsoluteQuantitationMethodFile.cpp

```diff
@@ -253,10 +148,34 @@ namespace OpenMS
       const Param tm_params = aqm.getTransformationModelParams();
       for (Size i = 0, j = 11; i < tm_params_names.size(); ++i, ++j)
       {
-        row[j] = tm_params.getValue(tm_params_names[i]);
+        row[j] = tm_params.exists(tm_params_names[i]) ? tm_params.getValue(tm_params_names[i]) : "";
```

Comment:
Does your unit tests pass with this line (e.g., if the parameter is missing)?  I would imagine if the type of the param is not compatible with an string it would throw an error.

Reply:
That's a good point. The tests pass but only because I'm testing that input and output files look the same.
I guess I should be checking the param name and lookup its type, and give a "default" value of `""` or `0` according to the found type.